modules/protocol/inspircd.c

```diff
@@ -10,6 +10,7 @@
 #include <atheme.h>
 #include <atheme/protocol/inspircd.h>
 
+// TODO: protocol 1204
```

Comment:
Protocols 1203 and 1204 never actually shipped and InspIRCd will not link against them. You can just support 1202 (v2) and 1205 (v3).

Reply:
I typo'd and meant 1205 lol